bin/ofs/tests/common/mod.rs

```diff
@@ -30,14 +27,16 @@ use tokio::runtime::{self};
 static INIT_LOGGER: OnceLock<()> = OnceLock::new();
 static RUNTIME: OnceLock<Runtime> = OnceLock::new();
 
+#[cfg(any(target_os = "linux", target_os = "freebsd"))]
```

Comment:
Mostly LGTM, but it seems we need to start to init windows environment test for ofs.

Reply:
`cloudfilter_opendal` and `fuse_opendal` provide different abilities to access remote, and `cloudfilter` needs to start a new process for testing, so I think we should rework the `ofs` behavior tests in next PR.

Reply:
Yeah, that's OK